tests/sentry/relay/config/test_metric_extraction.py

```diff
@@ -1110,40 +1106,27 @@ def test_get_metric_extraction_config_with_transactions_dataset(default_project:
         "count()", "transaction.duration:>=20", default_project, dataset=Dataset.Transactions
     )
 
-    # We test with prefilling, and we expect that both alerts are fetched since we support both datasets.
```

Comment:
Does this test make sense after removing the prefilling usage. Please let me know.

Reply:
we can remove all of this